lib/boilerplate.js

```diff
@@ -299,6 +300,10 @@ var boilerplate = function (gulp, opts) {
   });
 
   gulp.task('default', [opts.watch ? 'watch' : 'once']);
+
+  if (opts.preCommitTasks) {
+    gulp.task('pre-commit', opts.preCommitTasks);
```

Comment:
it might be nice to have a comment here along these lines:

```js
// this is a magic task that gets picked up by git-guppy automatically so we can run as a pre-commit git hook
```

because initially i was confused how this all linked into git-guppy and had to look at their docs to understand.